adapters/networkhandler/authConstants.js

```diff
@@ -3,5 +3,7 @@
  */
 module.exports = {
   DISABLE_DEST: "DISABLE_DESTINATION",
-  REFRESH_TOKEN: "REFRESH_TOKEN"
+  REFRESH_TOKEN: "REFRESH_TOKEN",
+  // To be used only for invalid/empty token error
+  INVALID_OR_EMPTY_TOKEN: "INVALID_OR_EMPTY_TOKEN"
```

Comment:
The first two constants indicate an action (disable or refresh), but this one doesn't fit.